src/framework/theme/components/radio/radio-group.component.ts

```diff
@@ -182,13 +182,11 @@ export class NbRadioGroupComponent implements AfterContentInit, OnDestroy, Contr
 
   writeValue(value: any): void {
     this.value = value;
-
-    if (typeof value !== 'undefined') {
-      this.updateValues();
-    }
+    this.updateValues();
   }
 
   protected updateAndSubscribeToRadios() {
+    this.getUpdateCheckedValue();
```

Comment:
Won't it override initially set `value` in the group component? There is a [PR](https://github.com/akveo/nebular/pull/841) which makes group component value take precedence over the radio component.